CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Compatibility Notes
 
+- OPA now requires Golang 1.13+ to build.
```

Comment:
The tester package too. I can back out using any of the custom metrics in the result from `opa test --bench` if we wanted to keep it only on `opa/cmd`.

Reply:
I suppose another way out would be adding helpers and using build flags to not support any of the custom metrics unless we're built with 1.13. I don't know how hard we want to pursue compat with older go versions, though.

Reply:
Yea, i started playing around with that. It gets kind of gross pretty quick.

IMO if we don't want to force 1.13 we should just drop the custom metrics until later, they're nice but not entirely crucial to this (we didn't know we could do it when originally planning this feature anyway).

Reply:
Hmm! I appreciate the histograms, but the standard metrics are a giant leap already. Since we've got the implementation and the docs already, cutting the custom metrics out feels a bit wasteful. 🤔

> I can back out using any of the custom metrics in the result from `opa test --bench`

Ah, you're suggesting to keep them in `opa bench` but remove them from `opa test`? That seems like a decent compromise...

Reply:
Yea, although it really only helps for anyone using the `tester` package who was _not_ using `cmd`.  Within OPA the only user of `tester` is `cmd/test.go` so it doesn't spread it out much. Anyone just using like the `rego` package (+dependencies it needs like `ast`/`topdown`/`storage`, etc) should be unaffected.

Reply:
The majority of library users are probably not dependent on `cmd` and `tester` so it's probably fine to leave the changes as is. I think that relaxing compatibility note might be enough (e.g., tell people that we use 1.13 now but if you aren't relying on cmd/tester you don't have to upgrade.)